core/src/fitting/kf_algorithm_defdet_cfield.cpp

```diff
@@ -0,0 +1,38 @@
+/** TRACCC library, part of the ACTS project (R&D line)
```

Comment:
I don't think defdet or teldet is very understandable to general users... how about 
`kalman_fitter_constant_field`
`kalman_fitter_constant_field_telescope` ?

Reply:
I don't have strong feelings about these file names. But keep in mind that users will not be exposed to any of these names. These are just implementation details behind the scenes. That's why I went with relatively cryptic names. Because people who are not developers of this code, would not be exposed to these files anyway.

Reply:
I believe having more verbosity is going to be helpful for devs as well.

Reply:
We will keep expanding the list of this specialization for different detector and inhomogeneous field
Being cryptic makes the maintenance more difficult I guess

Reply:
As I said, I'm very open to naming these files differently. I'll make this change shortly.